packages/components/pie-thumbnail/test/component/pie-thumbnail.spec.ts

```diff
@@ -1,7 +1,23 @@
 import { test, expect } from '@playwright/test';
 import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { getShadowElementStylePropValues } from '@justeattakeaway/pie-webc-testing';
+import { type ThumbnailProps } from '../../src/index.ts';
 
 const componentSelector = '[data-test-id="pie-thumbnail"]';
+const componentImgSelector = '[data-test-id="pie-thumbnail-img"]';
```

Comment:
Can these be added to a `selectors.ts` file in this directory like the other components?

Having a single definition will help to improve reusability of selectors, as eventually, we'll look to expose these to our consumers 👍🏼 

https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-card/test/helpers/page-object/selectors.ts